modules/ravestate_akinator/__init__.py

```diff
@@ -148,21 +141,34 @@ def is_it_answered(ctx):
             out = "Yeah! I guessed right! Thanks for playing with me! \nDo you want to play again?"
```

Comment:
The \n might be a bad idea for speech synthesis

Reply:
True :) it's gone